pyproject.toml

```
@@ -1,28 +1,91 @@
 [tool.poetry]
 name = "hack2022-dapla-hurtigstart"
-version = "1.0.0"
-description = ""
+version = "0.0.0"
```

It's best not to change this since the project is already established

```suggestion
version = "1.0.0"
```